experimental/packages/exporter-logs-otlp-http/package.json

```diff
@@ -74,6 +74,8 @@
     "@babel/core": "7.16.0",
     "@opentelemetry/api-logs": "0.40.0",
     "@types/mocha": "10.0.1",
+    "@types/mock-require": "^2.0.1",
+
```

Comment:
- dev dependencies version should be pinned.
- please remove the blank line.

Reply:
i just fixed the `package.json` :D